### PR TITLE
Add url to crx "file a bug"

### PR DESCRIPTION
**Pull request by ebidel**
R: all

We've seen an enormous uptick in bugs lately. Including the URL in the "file an issue" template will help us reduce the back and forth for debugging. Users can remove it before they file the issue.

### CODE REVIEW

File: lighthouse-extension/app/src/popup.js [hunk 1]
```diff
@@ -64,6 +66,11 @@ document.addEventListener('DOMContentLoaded', _ => {
 
     let qsBody = '**Lighthouse Version**: ' + getLighthouseVersion() + '\n';
     qsBody += '**Chrome Version**: ' + getChromeVersion() + '\n';
+
+    if (siteURL) {
+      qsBody += '**URL**: ' + siteURL + '\n';
```
Comment (patrickhulce) [inline lighthouse-extension/app/src/popup.js]: cool that `toString` of URL just gives you the url :) sensible!

**Comment by patrickhulce** [comment]
yes!! lgtm

**Comment by ebidel** [comment]
You gotta do the approval button for the PR to be mergeable :)

**Comment by patrickhulce** [comment]
first official approval, yay!

**Pull request merged**